media-libs/gmmlib/gmmlib-22.0.3-r1.ebuild

```diff
@@ -12,7 +12,7 @@ S="${WORKDIR}/${PN}-intel-${P}"
 
 KEYWORDS="~amd64"
 LICENSE="MIT"
-SLOT="0/12"
```

Comment:
I think we may want to do PV if they can't be trusted to bump SONAME.

Reply:
It's the first time when ABI changed without soname change for gmmlib. Most probably it was accidental.
It the moment I would say that this change is enough.

Reply:
OK. Please do setup iwdevtools and enable debugging symbols and its verbose abidiff output then to make sure such issues are noticed in future where possible. Thanks!